conf/app.routes

```diff
@@ -186,6 +186,11 @@ POST        /:srn/pension-commencement-lump-sum
 GET         /:srn/change-pension-commencement-lump-sum                                  controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumController.onPageLoad(srn: Srn, mode: Mode = CheckMode)
 POST        /:srn/change-pension-commencement-lump-sum                                  controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumController.onSubmit(srn: Srn, mode: Mode = CheckMode)
 
+GET         /:srn/pension-commencement-lump-sum-amount/:index                           controllers.nonsipp.memberreceivedpcls.PensionCommencementLumpSumAmountController.onPageLoad(srn: Srn, index: Max300, mode: Mode = NormalMode)
```

Comment:
Can go to pcls.routes https://github.com/hmrc/pension-scheme-return-frontend/blob/6ef70f53c1b72f2f667d14eb84c82a008a7fd131/conf/pcls.routes#L7